lib/Sema/SemaDecl.cpp

```diff
@@ -14205,8 +14214,10 @@ FieldDecl *Sema::ActOnField(Scope *S, Decl *TagD, SourceLocation DeclStart,
   FieldDecl *Res = HandleField(S, cast_or_null<RecordDecl>(TagD),
                                DeclStart, D, static_cast<Expr*>(BitfieldWidth),
                                /*InitStyle=*/ICIS_NoInit, AS_public);
-  if (S->isCheckedScope())
-    DiagnoseCheckedDecl(Res);
+  // Checked C - type restriction on declarations in checked block
```

Comment:
Some typos:
// Checked C - type restrictions on declarations in checked block.
// Member declaration is not allowed to use unchecked type in checked block.